components/event-publisher-proxy/pkg/handler/nats/handler.go

```diff
@@ -156,6 +156,7 @@ func (h *Handler) publishCloudEvents(writer http.ResponseWriter, request *http.R
 	}
 
 	h.receive(ctx, event)
+
```

Comment:
(nit) unnecessary changes in the file.

Reply:
I added intentionally to improve readability